Core/LocalAdmin.cs

```diff
@@ -181,7 +181,17 @@ internal async Task Start(string[] args)
         {
             await LoadJsonOrTerminate();
 
-            if (DataJson!.EulaAccepted == null)
+            var isEulaNotAccepted = DataJson!.EulaAccepted == null;
```

Comment:
Instead of using the `var` keyword, it would be better to ensure a `boolean` type. Since the output of `EulaAccepted` should theoretically be a boolean, no matter what. If it isn't, something is horribly wrong.

Reply:
Hi, I checked other parts of the code and saw that all variables are declared by `var`. Btw, Im using `is` prefix to define it as `boolean`

Reply:
> Instead of using the `var` keyword, it would be better to ensure a `boolean` type. Since the output of `EulaAccepted` should theoretically be a boolean, no matter what. If it isn't, something is horribly wrong.

C# is not Python or PHP. `var` doesn't mean that it can be sometimes bool, sometimes int and sometimes an array. It just lets you not define the type directly. It will be always bool, because `DataJson!.EulaAccepted == null` is always a bool (it's a comparison, it can't return anything other than a bool).

`var` is fine.

btw. `EulaAccepted` is a `DateTime?`, not a `bool`.